### PR TITLE
fix kick by laser

### PR DESCRIPTION
**Pull request by howard-stearns**
make ignore/kick work with hand controllers

Fixes https://highfidelity.fogbugz.com/f/cases/1836/Kick-and-Ignore-don-t-work-with-hand-lasers

### CODE REVIEW

**Comment by howard-stearns** [comment]
test plan; needs two avatars:
- bring two avatars to a domain.  Go into HMD in one of them.
- Use hand controllers in the HMD machine to click the ignore/kick button. (The labeling will be depend on whether you have kick permission in that domain.)
- An overlay should appear above the head of the other avatar. Use hand controllers to click that ignore/kick label. The avatar should go away.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 1051c068e145272d1612aabd4680d40ff124a96a available at: https://gist.github.com/5ccaaa48477f3cf02a6f4572ebe36d4c<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8771-1051c06.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8771-1051c06.dmg

File: scripts/system/mod.js [hunk 1]
```diff
@@ -193,7 +193,7 @@ var triggerMapping = Controller.newMapping(Script.resolvePath('') + '-click');
 function controllerComputePickRay(hand) {
     var controllerPose = getControllerWorldLocation(hand, true);
     if (controllerPose.valid) {
-        return { origin: controllerPose.position, direction: controllerPose.orientation };
+        return { origin: controllerPose.position, direction: Quat.getUp(controllerPose.orientation) };
```
Comment (Atlante45) [inline scripts/system/mod.js]: Up is forward for the controllers?

Comment (howard-stearns) [inline scripts/system/mod.js]: yeah, the controllers are weird.
my own reverse engineering model of what might be intended is that it follows the thumbs.
Anyway, up is correct.

**Comment by ZappoMan** [comment]
Tested - it works

**Pull request merged**